release/release-push

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh -euf
+#
+# release-push
```

Comment:
The scripts named 'release-xxxx' in this directory are meant to be used during the release process. Lets either rename this script, or move it up a level.

It's a bummer that we need such a complicated script for a simple push  task though. Oh well.